doc/user/clusterwide.md

```diff
@@ -0,0 +1,31 @@
+# Manage clusters in all namespaces
+
+Default etcd operator behavior is to only manage etcd clusters created in the same namespace.
+It is possible to deploy an etcd operator with special option to manage clusterwide etcd clusters.
+
+## Install etcd operator
+
+etcd operator have to run with `-all-namespaces` arg option.
+
+For example you can run:
```

Comment:
There is no need to mention the command and install guide here.
Just give a reference to the file.